xml/ay_bigfile.xml

```diff
@@ -4489,6 +4489,36 @@ config:type="boolean"&gt;true&lt;/diag&gt;</screen>
     <remark role="fixme">Add a short description</remark>
    </para>
 
+   <sect2 xml:id="Software.Selections.base_product">
+    <title>Product Selection</title>
+    <para>
+     Starting with SLE-15 all products are distributed using one media. You have to
```

Comment:
Use the `&slea;` entity here. Don't hyphenate "SLE 15".

Please use the singular of "media": "medium".

Maybe also make it clearer that there is a single medium for all products as opposed to the alternative reading that there might be one product-specific medium for each product (I hope that was understandable :/). E.g. change the sentence to something like: "Starting with &slea; 15, there is a single installation medium that allows installing all products."

Reply:
my TL told me that it can happen that there will be specific "one product" medium for some product(s) (at least opensuse and suse manager if I'm not wrong - @kobliha could you pls clarify if I'm wrong?)